### ISSUE TITLE
Recorder/Playback

### ISSUE THREAD

**Issue by tm1000** [jwetzell/showbridge#795]
Something that might be useful for Cue-View development is the ability to record bridges and then later play them back in real time

There's a few out there for OSC already

* https://pypi.org/project/osc-recorder/
* https://github.com/markcerqueira/oscreplayer
* https://github.com/trackme518/OSCreplay

**Comment by jwetzell** [comment]
I had actually had the thought of having an option to save messages to a file for logging purposes not necessarily for replay or anything but possible I guess. Messages of all types (HTTP is a little lossy) are pretty easily serializable because of the whole cloud portion which allows replicating messages to remote instances over the internet.

**Comment by jwetzell** [comment]
I'll add for debugging communication things I'm more of a `tcpdump` and `tcprelay` person myself.

**Comment by tm1000** [comment]
@jwetzell ah didn't even think about it at a tcp level

**Comment by jwetzell** [comment]
Yeah, while there is a good chunk of things that speak OSC, the difficult stuff tends to be the things that don't use it or the things that speak some weird non-compliant OSC that an actual OSC library hates to look at 🙃.

Although for the purposes of showbridge I think being able to record and playback messages might be interesting to explore. Maybe an action to start/stop recording and another action to playback a recording, although that might be a mess to try and illustrate to the user what is going on. Maybe it lives outside of the trigger->transform->action workflow.
